### PR TITLE
Move function definitions to source files

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: libs/crypto/include/crypto/bls_base.hpp [hunk 1]
```diff
@@ -56,46 +56,16 @@ enum
 
 void Init();
 
-inline PrivateKey PrivateKeyByCSPRNG()
-{
-  PrivateKey ret;
+PrivateKey PrivateKeyByCSPRNG();
```
Comment (ejfitzgerald) [inline libs/crypto/include/crypto/bls_base.hpp]: Can we revert this file

File: libs/ledger/include/ledger/chaincode/token_contract.hpp [hunk 2]
```diff
@@ -74,15 +74,5 @@ class TokenContract : public Contract
   StakeUpdates stake_updates_;
 };
 
-inline void TokenContract::ClearStakeUpdates()
```
Comment (ejfitzgerald) [inline libs/ledger/include/ledger/chaincode/token_contract.hpp]: Can we revert this file

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]

File: libs/ledger/include/ledger/chaincode/token_contract.hpp (hunk 2 above)
Comment (wkaluza) [reply libs/ledger/include/ledger/chaincode/token_contract.hpp]: Done

File: libs/crypto/include/crypto/bls_base.hpp (hunk 1 above)
Comment (wkaluza) [reply libs/crypto/include/crypto/bls_base.hpp]: Done

**Review by wkaluza — COMMENTED** [review]
Reverted

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**